packages/traceloop-sdk/traceloop/sdk/tracing/tracing.py

```diff
@@ -55,6 +55,7 @@ def __new__(
         processor: SpanProcessor = None,
         propagator: TextMapPropagator = None,
         exporter: SpanExporter = None,
+        instruments=[]
```

Comment:
Default value should be `None` otherwise you'll never go into the if at Line 113